runtime.md

```diff
@@ -34,21 +34,19 @@ See [Query State](#query-state) for information on retrieving the state of a con
 ## Lifecycle
 The lifecycle describes the timeline of events that happen from when a container is created to when it ceases to exist.
 
-1. OCI compliant runtime is invoked with a reference to the location of the bundle.
-   How this reference is passed to the runtime is an implementation detail.
-2. The container's runtime environment MUST be created according to the configuration in [`config.json`](config.md).
-   Any updates to `config.json` after container is running MUST not affect the container.
-3. The prestart hooks MUST be invoked by the runtime.
-   If any prestart hook fails, then the container MUST be stopped and the lifecycle continues at step 8.
-4. The user specified process MUST be executed in the container.
-5. The poststart hooks MUST be invoked by the runtime.
-   If any poststart hook fails, then the container MUST be stopped and the lifecycle continues at step 8.
-6. Additional actions such as pausing the container, resuming the container or signaling the container MAY be performed using the runtime interface.
+1. OCI compliant runtime's `create` command is invoked with a reference to the location of the bundle and a unique identifier.
+   How these references are passed to the runtime is an implementation detail.
+2. The container's runtime environment (namespaces, mounts, etc.) MUST be created according to the configuration in [`config.json`](config.md).
+   If a new PID namespace is requested in the [`config.json`](config.md), a PID namespace MUST created at this time.
+   However, the user-specified code (from ['process'](config.md#process-configuration) MUST NOT be executed at this time.
+   With the exception of the user-specified process, any updates to `config.json` after container is created MUST NOT affect the container.
+3. Runtime's `start` command is invoked with the unique identifier of the container.
+   The runtime MUST create and start the user-specified code, as specified by [`process`](config.md#process-configuration), in the container's PID namespace.
+   Any updates to the user-specified code in ['process'](config.md#process-configuration) after this point MUST NOT have any effect on the container.
+4. Additional actions such as pausing the container, resuming the container or signaling the container MAY be performed using the runtime interface.
```

Comment:
This can also happen before (3).  I think @julz made a comment to that effect at some point, but I can't dig up a reference now.  Some “MAY be invoked” wording in the `start` step, and some “this MAY happen before and/or after the `start` invocation” wording in this step should help clarify that.